README.md

```diff
@@ -11,68 +11,88 @@
+* Multiple implementations support: EmbedIO can use Microsoft `HttpListener` or internal Http Listener based on [Mono](https://www.mono-project.com/)/[websocket-sharp](https://github.com/sta/websocket-sharp/) projects
+* Cross-platform: tested on multiple OS and runtimes. From Windows .NET Framework to Linux MONO.
+* Extensible: Write your own modules -- For example, video streaming, UPnP, etc. Check out [EmbedIO Extras](https://github.com/unosquare/embedio-extras) for additional modules
 * Small memory footprint
 * Create REST APIs quickly with the out-of-the-box Web API module
-* Serve static files with 1 line of code (also out-of-the-box)
+* Serve static or embedded files with 1 line of code (also out-of-the-box)
 * Handle sessions with the built-in LocalSessionWebModule
-* WebSockets support (see notes below)
+* WebSockets support
 * CORS support. Origin, Header and Method validation with OPTIONS preflight
 * Supports HTTP 206 Partial Content
+* And many more options in the same package
+
+Some notes regarding WebSocket and runtimes support:
+
+| Runtime | HTTP implementation | WebSocket support | Notes |
+| --- | --- | --- | --- |
+| NET46 | *Unosquare* and Microsoft | Yes | Support Win7+ OS, Linux and macOS.  |
+| NET47 | Unosquare and *Microsoft* | | Yes | Support Win8+ OS. |
+| NETSTANDARD13 | *Unosquare* | Yes | Support Windows, Linux and macOS using native System.Net library |
+| NETSTANDARD20 | Unosquare and *Microsoft* | Yes | Support Windows, Linux and macOS using native System.Net library |
+| UAP | *Unosquare* | No | Support Windows Universal Platform. More information [here](https://github.com/unosquare/embedio/tree/master/src/Unosquare.Labs.EmbedIO.IoT) |
+
+### EmbedIO 2.0 - What's new
 
-*For detailed usage and REST API implementation, download the code and take a look at the Samples project*
+* `IHttpListener` is runtime/platform independent, you can choose Unosquare `HttpListener` implementation with NET47 or NETSTANDARD20. This separation of implementations brings new access to interfaces from common Http objects like `IHttpRequest`, `IHttpContext` and mor.
+* `IWebServer` is a new interface to create custom web server implementation, like a Test Web Server where all the operations are in-memory to speed up unit testing. Similar to [TestServer from OWIN](https://msdn.microsoft.com/en-us/library/microsoft.owin.testing.testserver(v=vs.113).aspx)
+* `WebApiController` is renewed. Reduce the methods overhead removing the WebServer and Context arguments. See example belows.
+* `RoutingStrategy.Regex` is the default routing scheme.
+* General improvements in how the Unosquare `HttpListner` is working and code clean-up.
 
-### **Some usage scenarios**:
+*Note* - We encourage to upgrade to new EmbedIO. Branch version 1.X will not be longer mantain, and issues will be tested against 2.X and resolved just there.
```

Comment:
1.  We encourage you to upgrade to the newest EmbedIO version ?
2.  Branch version 1.X **will no longer be** maintain**ed** (?)